src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,217 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.NoSuchElementException;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+    private static final Calculator calculator = new Calculator();
+    private static final double DELTA = 0.000001;
+
+    @Test
+    void additionWithTwoPositiveOperands() {
+        double actual = calculator.calculate(10, 20, Operations.PLUS);
+        double expected = 30;
+        assertEquals(expected, actual, DELTA);
+    }
+
+    @Test
+    void additionWithTwoNegativeOperands() {
```

Comment:
better add ok/notOk which behavior you expected
```suggestion
    void additionWithTwoNegativeOperands_Ok() {
```

Reply:
renamed all tests